sklearn/cluster/_affinity_propagation.py

```diff
@@ -32,7 +32,7 @@ def all_equal_similarities():
 
 def affinity_propagation(S, preference=None, convergence_iter=15, max_iter=200,
                          damping=0.5, copy=True, verbose=False,
-                         return_n_iter=False):
+                         return_n_iter=False, random_state=0):
```

Comment:
Shouldn't the default be `None` everywhere? Should it then go through a deprecation cycle?

Reply:
You are right: I have changed default values to `None` everywhere.... I'm a bit confused about deprecation... this is a new parameter, what exactly I have to deprecate? Thanks!